ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -107,6 +117,8 @@ def __init__(
         self._dry_run = dry_run
 
     def serialize(self) -> JsonObject:
+        self._serialized_files = ["nodes/*", "workflow.py", "state.py", "inputs.py", "display/*", "__init__.py"]
```

Comment:
set this field to an empty array in the init method and define it as a class attribute, similar to `_errors`